Assets/MRTK/Examples/Experimental/StepSlider/StepSliderExample.unity

```diff
@@ -0,0 +1,8073 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
+OcclusionCullingSettings:
```

Comment:
I'm getting an exception when I play this scene:

> Exception: Slider thumb on gameObject PinchSliderColorRed is not specified. Did you forget to set it?
Microsoft.MixedReality.Toolkit.UI.PinchSlider.Start () (at Assets/MRTK/SDK/Features/UX/Scripts/Sliders/PinchSlider.cs:255)